<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="2080" started-at="2020-04-21T00:30:34 IST" finished-at="2020-04-21T00:30:36 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="2080" started-at="2020-04-21T00:30:34 IST" finished-at="2020-04-21T00:30:36 IST">
      <class name="InterViewQuestions.alwaysRun">
        <test-method status="FAIL" signature="loginTest()[pri:0, instance:InterViewQuestions.alwaysRun@2b9627bc]" name="loginTest" duration-ms="2010" started-at="2020-04-21T00:30:34 IST" finished-at="2020-04-21T00:30:36 IST">
          <exception class="org.testng.internal.thread.ThreadTimeoutException">
            <message>
              <![CDATA[Method InterViewQuestions.alwaysRun.loginTest() didn't finish within the time-out 2000]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method InterViewQuestions.alwaysRun.loginTest() didn't finish within the time-out 2000
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:374)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:285)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:604)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.thread.ThreadTimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="VerifyTitle()[pri:0, instance:InterViewQuestions.alwaysRun@2b9627bc]" name="VerifyTitle" duration-ms="3" started-at="2020-04-21T00:30:36 IST" depends-on-methods="InterViewQuestions.alwaysRun.loginTest" finished-at="2020-04-21T00:30:36 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyTitle -->
      </class> <!-- InterViewQuestions.alwaysRun -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
